src/Service/Program.cs

```diff
@@ -23,10 +23,23 @@ public class Program
 
         public static void Main(string[] args)
         {
+            if (!ValidateAspNetCoreUrls())
+            {
+                Console.Error.WriteLine("Invalid ASPNETCORE_URLS format. See: https://github.com/Azure/data-api-builder/issues/2237");
```

Comment:
Also instead of issue, can we just put examples in the summary of the method:`ValidateAspNetCoreUrls`?